src/common/components/notification-modal/notificationModal-components/AlertCardModal.tsx

```diff
@@ -0,0 +1,64 @@
+import { useRef, useEffect } from "react";
+import classNames from "classnames/bind";
+import IC_CLOSE from "@/images/ic_close.svg";
+import GetAlertCardModalData from "@/shared/hooks/getAlertCardModalData";
+import CreateCards from "./CreateCards";
+import styles from "./AlertModal.module.scss";
+
+const cn = classNames.bind(styles);
+
+export interface AlertModalProps {
+  handleToggleModal: () => void;
```

Comment:
props로 받을때는 이렇게 표현해주는 편입니다~
```suggestion
  onToggleModal: () => void;
```